src/integrationTest/java/uk/gov/hmcts/darts/ws/AddAudioWebServiceTest.java

```diff
@@ -491,4 +492,66 @@ void testAddAudioHandleErrorInvalidSignature(DartsGatewayClient client) throws E
 
         }, DEFAULT_HEADER_USERNAME, DEFAULT_HEADER_PASSWORD);
     }
-}
+
+    @ParameterizedTest
+    @ArgumentsSource(DartsClientProvider.class)
+    void testAddAudioWithNoProblemEmptyResponse(DartsGatewayClient client) throws Exception {
+        authenticationStub.assertWithUserNameAndPasswordHeader(client, () -> {
+
+            String soapRequestStr = TestUtils.getContentsFromFile(
+                "payloads/addAudio/register/soapRequest.xml");
+
+            String dartsApiResponseStr = TestUtils.getContentsFromFile(
+                "payloads/addAudio/register/dartsApiResponse.json");
+
+            stubFor(post(urlPathEqualTo("/audios"))
+                        .willReturn(ok(dartsApiResponseStr).withStatus(500)));
+
+            XmlWithFileMultiPartRequest request = new DummyXmlWithFileMultiPartRequest(AddAudioMidTierCommand.SAMPLE_FILE);
+            when(requestHolder.getRequest()).thenReturn(Optional.of(request));
+
+            SoapAssertionUtil<AddAudioResponse> response = client.addAudio(getGatewayUri(), soapRequestStr);
+            Assertions.assertEquals("500", response.getResponse().getValue().getReturn().getCode());
+            Assertions.assertFalse(logAppender
+                                       .searchLogs(
+                                           AbstractClientProblemDecoder.RESPONSE_PREFIX +
+                                                       "500 Server Error: \"{<EOL>" +
+                                               "  \"code\": \"200\",<EOL>  \"message\": \"OK\"<EOL>}<EOL>\"", null, null).isEmpty());
```

Comment:
Why would we have a 500 server error with an message OK ?

Reply:
This is the way that the documentum interface works